server/repository/src/db_diesel/master_list.rs

```diff
@@ -65,6 +65,9 @@ impl<'a> MasterListRepository<'a> {
         let mut query = master_list_dsl::master_list.into_boxed();
 
         if let Some(f) = filter {
+            // Filter out inactive master lists by default
+            query = query.filter(master_list_dsl::is_active.eq(true));
```

Comment:
We have to always do it not only if the user specified a filter...

Reply:
[c3d8473](https://github.com/msupply-foundation/open-msupply/pull/2425/commits/c3d8473b7168b208651d37d22d5de5e273cb072f)